Basic/Basic/AppDelegate.m

```diff
@@ -9,41 +9,21 @@
 #import "AppDelegate.h"
 #import "AppDelegate.h"
 #import "MainViewController.h"
-#import "Employee.h"
-#import "Organization.h"
 #include "stdlib.h"
 
-@interface AppDelegate ()
+@interface AppDelegate () <UINavigationControllerDelegate>
 
 @end
 
 @implementation AppDelegate
 
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
-    // Override point for customization after application launch.
-        
-    Employee *empl = [[Employee alloc] initWithFirstName:@"Dima" lastName:@"Bondar" salary:1200];
     
-    NSLog(@"Full name: %@", [empl fullName]);
-    
-    Organization * organ = [[Organization alloc]initWithName:@"ww ee"];
-    
-    [organ addEmployeeWithName:@"tt rr"];
-    [organ addEmployeeWithName:@"tw rw"];
-    [organ addEmployeeWithName:@"tc rc"];
-    [organ addEmployee:empl];
-    
-    NSLog(@"Avarage salary: %f", [organ calculateAverageSalary]);
-    NSLog(@"LowestSalary: %i", [organ employeeWithLowestSalary]);
-    for (Employee * em in [organ employeesWithSalary:1300 tolerance:200]) {
-        NSLog(@"Tolerance: %i", em.salary);
-    }
-    
-    [organ removeEmployee:empl];
-    
-    NSLog(@"Avarage salary: %f", [organ calculateAverageSalary]);
-
+    UINavigationController *navigation = (UINavigationController *)self.window.rootViewController;
```

Comment:
You shouldn't do anything like that. To access managed object context, access AppDelegate's singleton ([UIApplication sharedApplication].delegate) and access the context directly